helm/defaultClusterIssuer/values.yaml

```diff
@@ -34,3 +34,15 @@ http:
 
     ingressLabels: {}
     ingressAnnotations: {}
+cloudflare:
+  cloudflare:
```

Comment:
```suggestion
  default:
```

Reply:
These values should be just an example, right? I think we should comment it out; otherwise, it will install this ClusterIssuer by default which is not desired.